services/app/src/components/base/DialogsContainer.tsx

```diff
@@ -27,7 +27,17 @@ const mapStateToProps = (state: AppState): StateProps => {
   };
 };
 
-const mapDispatchToProps = (dispatch: Redux.Dispatch<any>): DispatchProps => {
+function validateFeedback(text: string): Error|null {
+  if (!text) {
```

Comment:
We shouldn't block people exiting a quest if they don't want to type something - for example, someone could just be quickly trying out the quest to see if they like it

Reply:
We don't - this is used in `onFeedbackSubmit`, not in `onExitQuest`. I double-checked in a local instance that you don't need to give feedback to exit the quest. Also updated "submit" button to "exit" in the dialog so it makes more sense.